lib/main_ltp.pm

```diff
@@ -30,6 +31,10 @@ sub load_kernel_tests {
     loadtest_kernel "../installation/bootloader" if is_pvm;
 
     if (get_var('INSTALL_LTP')) {
+        if (is_alp) {
+            loadtest('microos/disk_boot');
+            loadtest('transactional/host_config');
+        }
         if (get_var('INSTALL_KOTD')) {
```

Comment:
Please add a blank line between the two `if` blocks so that it doesn't look like there should be an `else`. Otherwise LGTM.

Reply:
I thought that as well, but didn't want to bother with it.